reactive/etcd.py

```diff
@@ -155,30 +140,28 @@ def leader_config_changed():
             # Update the member's peer_urls with the new ports.
             log(etcdctl.member_update(members[unit_name]['unit_id'], url))
         # Render just the leaders configuration with the new values.
-        render('defaults', '/etc/default/etcd', bag.__dict__, owner='root',
-               group='root')
+        render_config()
```

Comment:
You are rendering configuration here and _not_ passing the bag variable. Did you mean to do that or not? The other uses appear to take bag as the